isb_lib/core.py

```diff
@@ -342,12 +344,15 @@ def solr_max_source_updated_time(
     }
     _url = f"{url}select"
     res = rsession.get(_url, headers=headers, params=params)
-    dict = res.json()
-    docs = dict["response"]["docs"]
-    if docs is not None and len(docs) > 0:
-        return dateparser.parse(docs[0]["sourceUpdatedTime"])
-    else:
-        return None
+    try:
```

Comment:
This is a bug that has always been here if you were to try and run against an empty index.  This was added after we had data in the index, so we never hit it before.